HW4RonBitonPF.py

```diff
@@ -0,0 +1,9 @@
+from collections import Counter
```

Comment:
please do not use this module, we are required to write your own algorithm to solve the challenge